include/mgp.hpp

```diff
@@ -596,10 +562,11 @@ class Node {
   bool HasLabel(std::string_view label) const;
 
   /// @brief Returns an iterable & indexable structure of the node’s properties.
-  class Properties Properties() const;
+  std::map<std::string, Value> Properties() const;
 
-  /// @brief Returns the value of the node’s `property_name` property.
-  Value operator[](const std::string_view property_name) const;
+  void SetProperty(std::string property, Value value) const;
```

Comment:
```suggestion
  void SetProperty(std::string property, Value value);
```

Even though the underlying implementation can be `const` because of the operation doesn't change this object, logically this shouldn't be const. This objects represents a vertex which is changed by modifying one of its properties.

Reply:
I agree, changed it!